x-pack/plugins/streams_app/public/components/stream_detail_view/index.tsx

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EntityDetailViewWithoutParams, EntityViewTab } from '../entity_detail_view';
+import { useStreamsAppParams } from '../../hooks/use_streams_app_params';
+import { useStreamsAppFetch } from '../../hooks/use_streams_app_fetch';
+import { useKibana } from '../../hooks/use_kibana';
+import { StreamDetailOverview } from '../stream_detail_overview';
+
+export function StreamDetailView() {
+  const {
+    path: { key, tab },
+  } = useStreamsAppParams('/{key}/{tab}');
+
+  const {
+    dependencies: {
+      start: {
+        streams: { streamsRepositoryClient },
+      },
+    },
+  } = useKibana();
+
+  const { value: streamEntity } = useStreamsAppFetch(
+    ({ signal }) => {
+      return streamsRepositoryClient.fetch('GET /api/streams/{id}', {
+        signal,
+        params: {
+          path: {
+            id: key,
+          },
+        },
+      });
+    },
+    [streamsRepositoryClient, key]
+  );
+
+  const entity = {
+    id: key,
+    displayName: key,
+  };
+
+  const tabs: EntityViewTab[] = [
+    {
+      name: 'overview',
+      content: <StreamDetailOverview definition={streamEntity} />,
+      label: i18n.translate('app_not_found_in_i18nrc.streamDetailView.overviewTab', {
+        defaultMessage: 'Overview',
+      }),
+    },
+    {
```

Comment:
I thought we would go with a single management tab which has a second level of navigation for the individual things?

Reply:
yh, just placeholders, will change in this PR